### PR TITLE
Added tabs functionality, will fix size later

### PR DESCRIPTION
**Pull request by atvick21**

### CODE REVIEW

**Review by calebd-anderson — APPROVED** [review]

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,8 +1,3 @@
 <div class="container">
-    <mat-toolbar color="warn">
-        <span class="fill-remaining-space"></span>
-    </mat-toolbar>
-</div>
-<router-outlet></router-outlet>
-
-<notifier-container></notifier-container>
```
Comment (calebTree) [inline src/app/app.component.html:8]: not sure why `notifier-container` is removed

File: src/app/tabs/tabs.component.html [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
     <a mat-tab-link
             *ngFor="let link of tabLinks"
             [routerLink]="link.path"
             routerLinkActive #rla="routerLinkActive"
-            [active]="rla.isActive"> {{link.label}}
+            [active]="rla.isActive"> 
+            {{link.label}}
     </a>
 </nav>
 <router-outlet></router-outlet>
```
Comment (calebTree) [inline src/app/tabs/tabs.component.html:10]: router-outlet unnecessary here

**Review by calebd-anderson — CHANGES_REQUESTED** [review]
I'm clumsy with GitHub reviewing.

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,8 +1,3 @@
 <div class="container">
-    <mat-toolbar color="warn">
-        <span class="fill-remaining-space"></span>
-    </mat-toolbar>
-</div>
-<router-outlet></router-outlet>
```
Comment (calebTree) [inline src/app/app.component.html:6]: `router-outlet` could stay here

**Review by calebTree — CHANGES_REQUESTED** [review]

File: src/app/app.component.html (hunk 3 above)
Comment (calebTree) [inline src/app/app.component.html:6]: `router-outlet` could stay here

Comment (calebd-anderson) [inline src/app/app.component.html:6]: `<router-outlet>` could stay here.

File: src/app/tabs/tabs.component.html (hunk 2 above)
Comment (calebd-anderson) [inline src/app/tabs/tabs.component.html:10]: `<router-outlet>` unnecessary here.

File: src/app/app.component.html (hunk 1 above)
Comment (calebd-anderson) [inline src/app/app.component.html:8]: Not sure why `<notifier-container>` is removed here?

**Review by calebd-anderson — CHANGES_REQUESTED** [review]
Not sure if you're going to keep working on this? We could close it and review #15 instead.

**Comment by calebd-anderson** [comment]
Actually, #15 fixes the login page kinda out of whack so we might as well just look at that PR.

**Pull request closed without merging**